Makefile

```diff
@@ -35,39 +43,34 @@ clean:
 .PHONY: clean
 
 devnet-up:
-	@bash ./ops/devnet-up.sh
+	@bash ./ops-bedrock/devnet-up.sh
```

Comment:
If we are going to be putting this on a branch of the monorepo, should this be `ops` still?

Reply:
I think we should separate them for a bit. This lets us incrementally merge things together.